build/media_src/system/js/multiselect.es6.js

```diff
@@ -71,7 +71,7 @@
 
           this.boxes[currentCheckBox].checked = !this.boxes[currentCheckBox].checked;
           isChecked = this.boxes[currentCheckBox].checked;
-          Joomla.isChecked(this.boxes[currentCheckBox].checked);
+          Joomla.isChecked(this.boxes[currentCheckBox].checked, this.tableEl.id);
```

Comment:
Can you please do a PR with this change in the main repo?

Reply:
> Can you please do a PR with this change in the main repo?

@dgrammatiko This will create issue. because some more files is changed to adopt this change.
So, i think better to merge here